buy_together_app/models.py

```diff
@@ -18,14 +18,16 @@ class User(models.Model):
                                  validators=[MinLengthValidator(6)],
                                  help_text="User Name")
     first_name = models.CharField(max_length=32,
+                                  null=True,
```

Comment:
Are we allowing no value for this field (and the ones below)?

Reply:
No it has minvalifator and in the frontend the form will maych the validator this is done like that to pass a warning when migration created

Reply:
ok. Looking at Django docs, this seems to be a bad practice. https://docs.djangoproject.com/en/4.1/ref/models/fields/#null

Reply:
Fixed!